### PR TITLE
Add new trigger after ".sonata-collection-row" successfully removed.

### PR DESCRIPTION
**Pull request by tim96**
Add new trigger event, when ".sonata-collection-row" successfully removed. Issue #3442

### CODE REVIEW

File: Resources/public/Admin.js [hunk 1]
```diff
@@ -358,6 +358,8 @@ var Admin = {
             jQuery(this).trigger('sonata-collection-item-deleted');
 
             jQuery(this).closest('.sonata-collection-row').remove();
+
+            jQuery(this).trigger('sonata-collection-item-deleted-successfully');
```
Comment (OskarStark) [inline Resources/public/Admin.js]: successfull< -> successfull

Comment (tim96) [inline Resources/public/Admin.js]: > successfully -> successfull ?

fixed

File: Resources/public/Admin.js [hunk 2]
```diff
@@ -358,6 +358,8 @@ var Admin = {
             jQuery(this).trigger('sonata-collection-item-deleted');
 
             jQuery(this).closest('.sonata-collection-row').remove();
+
+            jQuery(this).trigger('sonata-collection-item-deleted-successfull');
```
Comment (javiereguiluz) [inline Resources/public/Admin.js]: Actually, in English `successfull` is spelled `successful`

Comment (OskarStark) [inline Resources/public/Admin.js]: :+1:

Comment (tim96) [inline Resources/public/Admin.js]: @OskarStark done.

**Comment by OskarStark** [comment]
@tim96 please squash your commits and add this new event to the documentation with an example, if possible.

thank you

**Comment by tim96** [comment]
@OskarStark fixed. What type of example I need add. Something like that:
```
// subscribe to event
$(document).on('sonata-collection-item-deleted-successful', function() {
// call after a row has been deleted
});
```

**Comment by OskarStark** [comment]
no i mean an example, for what you can use it.

**Comment by tim96** [comment]
@OskarStark hello. I have a lot of collection in my form, so I have plan to using trigger ``sonata-collection-item-deleted`` for ask user before delete row, and using triggers ``sonata-collection-item-added`` and ``sonata-collection-item-deleted-successful`` for check state of collection after row was added or deleted, and maybe anybody have such tasks.

**Comment by OskarStark** [comment]
Thank you @tim96

**Comment by BenoitLeveque** [comment]
@OskarStark  @tim96 hey this can't work as the element as be deleted from the dom

**Comment by tim96** [comment]
@BenoitLeveque yes, you are right, my fault((( I'm sorry, I can't remember why I using ``this`` instead of ``document`` , in documentation I using ``document``. @OskarStark can we add @BenoitLeveque PR, please?

**Comment by OskarStark** [comment]
already done @tim96 :smile: 

thank you!

**Pull request merged**